core-library/src/main/java/org/silverpeas/core/admin/domain/DomainDriver.java

```diff
@@ -35,107 +36,115 @@
-  /**
-   * Create GroupDetail action Mask
-   * @see #getDriverActions
-   */
-  long ACTION_CREATE_GROUP = 0x00000020;
-  /**
-   * Delete User action Mask
-   * @see #getDriverActions
-   */
-  long ACTION_DELETE_USER = 0x00000040;
-  /**
-   * Delete GroupDetail action Mask
-   * @see #getDriverActions
-   */
-  long ACTION_DELETE_GROUP = 0x00000080;
-  /**
-   * Add/Remove User from group action Mask
-   * @see #getDriverActions
-   */
-  long ACTION_EDIT_USER_IN_GROUP = 0x00000100;
-  /**
-   * Add a user in Silverpeas DB by synchronization with a reference LDAP DB
-   * @see #getDriverActions
-   */
-  long ACTION_IMPORT_USER = 0x00000200;
-  /**
-   * Updates user Silverpeas infos from LDAP DB
-   * @see #getDriverActions
-   */
-  long ACTION_SYNCHRO_USER = 0x00000400;
-  /**
-   * Remove user entry from Silverpeas
-   * @see #getDriverActions
-   */
-  long ACTION_REMOVE_USER = 0x00000800;
-  /**
-   * Add a group in Silverpeas DB by synchronization with a reference LDAP DB
-   * @see #getDriverActions
-   */
-  long ACTION_IMPORT_GROUP = 0x00001000;
-  /**
-   * Updates group Silverpeas infos from LDAP DB
-   * @see #getDriverActions
-   */
-  long ACTION_SYNCHRO_GROUP = 0x00002000;
-  /**
-   * Remove group entry from Silverpeas
-   * @see #getDriverActions
-   */
-  long ACTION_REMOVE_GROUP = 0x00004000;
-  /**
-   * Create a x509 certificate and store it in server's truststore
-   * @see #getDriverActions
-   */
-  long ACTION_X509_USER = 0x00008000;
-  /**
-   * All available actions Mask
-   * @see #getDriverActions
-   */
-  long ACTION_MASK_ALL = 0xFFFFFFFF;
-  long ACTION_MASK_RW = ACTION_READ_USER |
-      ACTION_READ_GROUP | ACTION_UPDATE_USER | ACTION_UPDATE_GROUP |
-      ACTION_CREATE_USER | ACTION_CREATE_GROUP | ACTION_DELETE_USER |
-      ACTION_DELETE_GROUP | ACTION_EDIT_USER_IN_GROUP;
-  long ACTION_MASK_RO = ACTION_READ_USER | ACTION_READ_GROUP | ACTION_IMPORT_USER
-      | ACTION_SYNCHRO_USER | ACTION_REMOVE_USER | ACTION_IMPORT_GROUP | ACTION_SYNCHRO_GROUP |
-      ACTION_REMOVE_GROUP;
-  long ACTION_MASK_MIXED_GROUPS = ACTION_READ_GROUP | ACTION_UPDATE_GROUP | ACTION_CREATE_GROUP
-      | ACTION_DELETE_GROUP | ACTION_EDIT_USER_IN_GROUP;
-
-  void init(int domainId, String initParam, String authenticationServer) throws Exception;
+  class ActionConstants {
+
+    private ActionConstants() {
+
+    }
+
+    /**
+     * No possible actions Mask
+     * @see #getDriverActions
+     */
+    public static final long ACTION_NONE = 0x00000000;
+    /**
+     * Read Users' infos action Mask
+     * @see #getDriverActions
+     */
+    public static final long ACTION_READ_USER = 0x00000001;
+    /**
+     * Read Groups' infos action Mask
+     * @see #getDriverActions
+     */
+    public static final long ACTION_READ_GROUP = 0x00000002;
+    /**
+     * Update Users' infos action Mask
+     * @see #getDriverActions
+     */
+    public static final long ACTION_UPDATE_USER = 0x00000004;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)